src/app.tsx

```diff
@@ -64,7 +64,7 @@ export const App = () => {
 	return (
 		<BrowserRouter>
 			<AKRoutes>
-				<Route element={<AppLayout />} path="/">
+				<Route element={<AppLayout topbarHidden={true} />} path="/">
```

Comment:
I think that it should be enough:
```suggestion
				<Route element={<AppLayout topbarHidden />} path="/">
```

Reply:
Also please rename the property to: `hideTopbar`